pkg/objstore/objtesting/foreach.go

```diff
@@ -18,23 +20,38 @@ import (
 	"github.com/thanos-io/thanos/pkg/testutil"
 )
 
+// IsObjStoreSkipped returns true if given provider ID is found in THANOS_TEST_OBJSTORE_BLACKLIST array delimited by comma e.g:
+// THANOS_TEST_OBJSTORE_BLACKLIST=GCS,S3,AZURE,SWIFT,COS,ALIYUNOSS
+func IsObjStoreSkipped(t *testing.T, provider client.ObjProvider) bool {
+	if e, ok := os.LookupEnv("THANOS_TEST_OBJSTORE_BLACKLIST"); !ok {
```

Comment:
Naming nit: can we call this instead `THANOS_TEST_OBJSTORE_SKIP`? In general I think we’d like to avoid using blacklist and whitelist

Reply:
It's a list though (: Maybe `skip list`?

Reply:
Sure but in spoken English you could say “please skip tests A B and C” there’s no need for the word list IMO. I think this is also common in Unix tools. Plus, why make the flag name any longer and harder to type :p

Reply:
fair, done (: